src/lib.rs

```diff
@@ -1,4 +1,6 @@
 use std::collections::HashMap;
+use std::collections::hash_map::Iter;
+use std::collections::hash_map::IterMut;
```

Comment:
Should these at least be `pub use` as a step towards making `HashMap` an implementation detail?

More ideally would be for a newtype but reimplementing the interface is a bit annoying.